orquesta/events.py

```diff
@@ -70,6 +72,8 @@
     WORKFLOW_PAUSED_WORKFLOW_DORMANT,
     WORKFLOW_RESUMING,
     WORKFLOW_RESUMING_WORKFLOW_COMPLETED,
+    WORKFLOW_RERUNNING,
+    WORKFLOW_RERUNNING_WORKFLOW_COMPLETED,
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.